source/unified-test-format/tests/Makefile

```
@@ -1,48 +1,54 @@
 SCHEMA=../schema-1.23.json
 
-.PHONY: all invalid valid-fail valid-pass atlas-data-lake versioned-api load-balancers gridfs transactions transactions-convenient-api crud collection-management read-write-concern retryable-reads retryable-writes sessions command-logging-and-monitoring client-side-operations-timeout HAS_AJV
+.PHONY: all atlas-data-lake auth change-streams client-side-encryption client-side-operations-timeout collection-management command-logging-and-monitoring command-logging-and-monitoring/logging command-logging-and-monitoring/monitoring connection-monitoring-and-pooling connection-monitoring-and-pooling/logging crud gridfs index-management load-balancers read-write-concern retryable-reads retryable-writes run-command server-discovery-and-monitoring server-selection server-selection/logging sessions transactions-convenient-api transactions unified-test-format unified-test-format/invalid unified-test-format/valid-fail unified-test-format/valid-pass versioned-api HAS_AJV
```

[optiona] split target list for readability?

```suggestion
.PHONY: all \  
        atlas-data-lake \  
        auth \  
        change-streams \  
        client-side-encryption \  
        client-side-operations-timeout \  
        collection-management \  
        command-logging-and-monitoring \  
        command-logging-and-monitoring/logging \  
        command-logging-and-monitoring/monitoring \  
        connection-monitoring-and-pooling \  
        connection-monitoring-and-pooling/logging \  
        crud \  
        gridfs \  
        index-management \  
        load-balancers \  
        read-write-concern \  
        retryable-reads \  
        retryable-writes \  
        run-command \  
        server-discovery-and-monitoring \  
        server-selection \  
        server-selection/logging \  
        sessions \  
        transactions-convenient-api \  
        transactions \  
        unified-test-format \  
        unified-test-format/invalid \  
        unified-test-format/valid-fail \  
        unified-test-format/valid-pass \  
        versioned-api \  
        HAS_AJV  

```

---

Changed